lib/gettext-setup/pot.rb

```diff
@@ -137,5 +137,23 @@ def self.update_pot(locales_path = GettextSetup.locales_path, path = nil)
         end
       end
     end
+
+    def self.merge_pot(locales_path = GettextSetup.locales_path)
+      mergepot = GettextSetup.config['project_name'] + '.pot'
+      if !File.exist? mergepot
```

Comment:
Not sure this is correct, as the GettextSetup.config['project_name'] + '.pot' will always exist. 
It should probably overwrite what is there.  Deleting the main pot file over and over is a pain.